data_dir/cs_profile_background_reads_overload.yaml

```diff
@@ -0,0 +1,39 @@
+keyspace: keyspace1
```

Comment:
Where did you get this profile from?
why not using an existing one?

Reply:
all other existing profiles use replication_factor >= 2 (and as i have a single db node, it must be 1)
the only one i found that uses replication_factor is the default, and i didn't try it, i created this one based on existing ones.